cli/config/configuration-acrn.toml.in

```diff
@@ -16,6 +16,11 @@ ctlpath = "@ACRNCTLPATH@"
 kernel = "@KERNELPATH_ACRN@"
 image = "@IMAGEPATH@"
 
+# List of valid annotation names for the hypervisor
+# Each member of the list is a regular expression, which is the base name
+# of the annotation, e.g. "path" for io.katacontainers.config.hypervisor.path"
+enable_annotations = @DEFENABLEANNOTATIONS@
```

Comment:
Another question I have is I only see this for hypervisor specific section here, so are annotations related to the runtime/agent affected by this change?

Reply:
(Reposting comment, which for some reason I attached to your previous comment)

> Another question I have is I only see this for hypervisor specific section here, so are annotations related to the runtime/agent affected by this change?

No. Annotations in the agent cannot cause the kind of damage the runtime can. See the discussion on Launchpad.

Reply:
Again, I asked because in the discussion I was under the impression that we talked about enabling/disabling all annotations and not hypervisor related annotations.

Reply:
In the long run, yes. It's less important and urgent, however.